packages/test/test-end-to-end-tests/src/test/gc/gcSweepUnreferencePhases.spec.ts

```diff
@@ -126,83 +126,82 @@ describeCompat("GC sweep unreference phases", "NoCompat", (getTestObjectProvider
 		);
+
+		// Wait the other half of sweepTimeoutMs, triggering Tombstone
+		await delay(sweepTimeoutMs / 2);
+		mainDataStore._root.set("send", "op2");
+		await provider.ensureSynchronized();
+		summaryTree = (await summarizeNow(summarizer)).summaryTree;
+
+		const rootGCTree = summaryTree.tree[gcTreeKey];
+		assert.equal(rootGCTree?.type, SummaryType.Tree, "GC data should be a tree");
+		tombstoneState = getGCTombstoneStateFromSummary(summaryTree);
+		// After sweepTimeoutMs the object should be tombstoned.
+		assert(tombstoneState !== undefined, "Should have tombstone state");
 		assert(
-			await isDataStoreInSummaryTree(summaryTree2, dataStoreId),
-			"Data Store should be in the summary!",
+			tombstoneState.includes(dataStoreHandle.absolutePath),
+			"Datastore should be tombstoned",
 		);
 
-		// Wait sweep timeout
-		await delay(sweepTimeoutMs);
+		// Wait tombstoneSweepDelayMs, triggering Sweep
+		await delay(tombstoneSweepDelayMs);
 		mainDataStore._root.set("send", "op2");
 		await provider.ensureSynchronized();
-		const summary3 = await summarizeNow(summarizer);
-		const summaryTree3 = summary3.summaryTree;
+		const summaryWithObjectDeleted = await summarizeNow(summarizer);
+		summaryTree = summaryWithObjectDeleted.summaryTree;
 		// GC graph check
-		const gcState3 = getGCStateFromSummary(summaryTree3);
+		const gcState3 = getGCStateFromSummary(summaryTree);
 		assert(gcState3 !== undefined, "Expected GC state to be generated");
 		assert(
 			!(dataStoreHandle.absolutePath in gcState3.gcNodes),
 			"Data Store should not exist on gc graph",
 		);
 		// GC Tombstone check
-		const tombstoneState3 = getGCTombstoneStateFromSummary(summaryTree3);
-		assert(tombstoneState3 === undefined, "Nothing should be tombstoned");
+		tombstoneState = getGCTombstoneStateFromSummary(summaryTree);
+		assert(tombstoneState === undefined, "Nothing should be tombstoned");
 		// GC Sweep check
-		const deletedState3 = getGCDeletedStateFromSummary(summaryTree3);
-		assert(deletedState3 !== undefined, "Should have sweep state");
-		assert(deletedState3.includes(dataStoreHandle.absolutePath), "Data Store should be swept");
-		assert(deletedState3.includes(ddsHandle.absolutePath), "DDS should be swept");
-		assert(deletedState3.length === 2, "Nothing else should have been swept");
+		deletedState = getGCDeletedStateFromSummary(summaryTree);
+		assert(deletedState !== undefined, "Should have sweep state");
+		assert(deletedState.includes(dataStoreHandle.absolutePath), "Data Store should be swept");
+		assert(deletedState.includes(ddsHandle.absolutePath), "DDS should be swept");
+		assert(deletedState.length === 2, "Nothing else should have been swept");
 		// Summary check
 		assert(
-			!(await isDataStoreInSummaryTree(summaryTree3, dataStoreId)),
+			!(await isDataStoreInSummaryTree(summaryTree, dataStoreId)),
 			"Data Store should not be in the summary!",
 		);
 
-		await provider.loadTestContainer(testContainerConfig, {
```

Comment:
To my eyes, this wasn't doing anything.  And the tests pass without it.  If I'm missing something please fill me in!  cc @tyler-cai-microsoft

Reply:
This just validates that a remote container that doesn't "do" anything doesn't blow up. If we get that coverage anyway, then we're fine here.